core/js/storages/localstorageService.js

```diff
@@ -1,261 +0,0 @@
-'use strict';
```

Comment:
In `ev-fdm` no. And on our own codebase (_because I guess that it was your question purpose_) neither.
Maybe someone using `ev-fdm` may need it, but I'm not quite sure.